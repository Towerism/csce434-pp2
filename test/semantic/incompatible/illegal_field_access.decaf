class Asdf {
      int a;
}

int asdf() {
    Asdf b;
    b.a;
}