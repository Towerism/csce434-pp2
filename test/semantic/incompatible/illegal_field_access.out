
*** Error line 7.
    b.a;
      ^
*** Asdf field 'a' only accessible within class scope

